AddItem.ts

```diff
@@ -8,16 +9,20 @@ const db = new sqlite3.Database("./db/items.db", error => {
 });
 
 export async function AddNewItem(
-    itemID: number,
-    itemName: string,
-    itemPrice: string,
-    itemDesc: string,
+    item_name: string,
+    description: string,
+    top_bidder: string,
+    price: number,
+    shipping_cost: number,
+    active: boolean,
+    auction_type: auction_type,
+    end_time: Date
 ): Promise<number> {
 
     return new Promise((resolve,reject) => {
         db.run(
-            "INSERT INTO items (itemID, itemName, itemPrice, itemDesc) VALUES ($1,$2,$3,$4)",
-            [itemID, itemName, itemPrice, itemDesc],
+            "INSERT INTO items (item_name, description, top_bidder, price, shipping_cost, active, auction_type, end_time) VALUES ($1,$2,$3)",
```

Comment:
Should it not be more than `VALUES ($1,$2,$3)`? since you now have like 8 values